src/main/java/org/apache/commons/lang3/function/Functions.java

```diff
@@ -41,4 +41,18 @@ public static <T, R> Function<T, R> function(final Function<T, R> function) {
     private Functions() {
         // no instances needed.
     }
+
+    /**
+     * Applies the {@link Function} on the object if the object is not {@code null}. If the object is {@code null}, it
+     * does nothing.
+     *
+     * @param object the object to apply the function.
+     * @param function the function to apply.
+     * @param <T> the type of the argument the function applies.
+     * @param <R> the type of the result the function returns.
+     * @return the value the function returns If the object is null; null otherwise.
```

Comment:
- The name can just be `apply`.
- Add a Javadoc since tag.
- Add null-check for `function`, see `Suppliers.get()`